docker/production/docker-compose.production.yml

```diff
@@ -85,16 +99,17 @@ volumes:
   media:
     driver_opts:
       type: "nfs"
-      o: "addr=krikkit,noatime,nolock,soft,vers=4.2,rw"
+      o: "addr=yourfileserverdomain,noatime,nolock,soft,vers=4.2,rw"
       device: ":/mampf/media/mampf"
   submissions:
     driver_opts:
       type: "nfs"
-      o: "addr=krikkit,noatime,nolock,soft,vers=4.2,rw"
+      o: "addr=yourfileserverdomain,noatime,nolock,soft,vers=4.2,rw"
       device: ":/mampf/submissions/mampf"
   public:
+  nginx-conf-d:
   caches:
     driver_opts:
       type: "nfs"
-      o: "addr=krikkit,noatime,nolock,soft,vers=4.2,rw"
+      o: "addr=yourfileserverdomain,noatime,nolock,soft,vers=4.2,rw"
       device: ":/mampf/caches/mampf"
```

Comment:
maybe add some infos about this into `INSTALL.md`

Reply:
Done.